src/model/routes/modelRouter.ts

```diff
@@ -7,6 +7,8 @@ const modelRouterFactory: FactoryFunction<Router> = (dependencyContainer) => {
   const controller = dependencyContainer.resolve(ModelController);
 
   router.post('/', controller.createModel);
+  router.post('/validate', controller.validate.bind(controller));
+  router.post('/validateSources', controller.validateSources.bind(controller));
```

Comment:
Consider change the route
```suggestion
  router.post('/validate/sources', controller.validateSources.bind(controller));
```

Reply:
why? i think it's better not to put both validation in same hirerachy

Reply:
Why not? nevertheless, it is not the best practice to use camel case in URIs.
See: https://restfulapi.net/resource-naming/